doc/method/results.md

```diff
@@ -1,11 +1,16 @@
+## Summary of Open Tree Taxonomy
 
-## Homonym analysis
+The methods and results presented here are for version 3.0 of the Open Tree Taxonomy (which follows five previous releases using the automated assembly method). The taxonomy contains 3,592,827 total taxa, 3,271,403 tips and 276,375 internal nodes. 2,335,143 of the nodes have a Linnean binomial of the form Genus epithet. There are 2,027,352 synonym records and 8154 homonyms (name-strings for which there are multiple nodes). A longer list of metrics is in supplementary data.
 
-There are 8154 name-strings in the final version of the taxonomy for which there are
-multiple nodes.  <!-- [JAR: update number when 3.0 is final] -->
-By comparison, there are only 1440 in GBIF. Many of
-the homonyms either are artifacts of the alignment method, or reflect
-misclassifications or errors in the source taxonomies.
+## Results of assembly procedure
+
+As OTT is assembled, the alignment procedure processes every source node, either choosing an alignment target for it in the workspace based on the results of the heuristics, or leaving it unaligned. Figure 4 illustrates the results of the alignment phase. We note that presence of a single candidate node does not automatically align the two nodes - we still apply the heuristics to ensure a match (and occasionally reject the single candidate).  
+
+When reducing a candidate set from multiple nodes to one node, we counted the frequency of success for each heuristic, i.e. the number of times that a particular heuristic was the one that accepted a single candidate. Table X shows these results. Separation (do not align taxa in different groups, according to the separation taxonomy; used first), Lineage (align taxa with shared lineage; used midway through) and Same-name-string (prefer candidates who primary name string matches; used last) were by far the most frequent.
```

Comment:
number of times that a particular heuristic was the one that accepted the winning candidate from among two or more
[i.e.: the counts do not include the situations where there was only one candidate and that candidate was confirmed by the given heuristic]